packages/tables/resources/views/columns/text-column.blade.php

```diff
@@ -178,10 +178,15 @@
                         ])
                     >
                         @if ($isBadge)
+                            @php
+                                $badgeTooltip = $getBadgeTooltip($state);
```

Comment:
No need to store in a variable if it's only used once.

Reply:
Fixed!